apps/evm/lib/evm/builtin.ex

```diff
@@ -22,6 +21,8 @@ defmodule EVM.Builtin do
   @g_quaddivisor 20
   @g_ec_add 500
   @g_ec_mult 40_000
+  @g_ec_pairing_point 80_000
+  @g_ec_pairing 100_000
```

Comment:
Are these pairing points specified in the EIP? Just curious here, not asking for a change.